core-java-modules/core-java-datetime-string-2/src/test/java/com/baeldung/timestamp/CurrentTimeAsFileNameUnitTest.java

```diff
@@ -0,0 +1,104 @@
+package com.baeldung.timestamp;
+
+import static org.junit.Assert.assertTrue;
+
+import java.text.MessageFormat;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.regex.Pattern;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.junit.Test;
+
+public class CurrentTimeAsFileNameUnitTest {
+    private static final String TIMESTAMP_FORMAT = "yyyyMMddHHmmss";
```

Comment:
- Could we add a space between the top of the class and the constant declarations? It makes the class slightly easier to read.

- Could we also add a space between test setup and assertions? For example,
  ```
  @Test
    public void whenUsingDate_thenGetCurrentTime() {
        String currentTime = SIMPLEDATE_FORMATTER.format(new Date());
        String fileName = getFileName(currentTime);

        assertTrue(verifyFileName(fileName));
    }
```